packages/entityanalytics_okta/data_stream/user/elasticsearch/ingest_pipeline/default.yml

```
@@ -5,6 +5,20 @@ processors:
       field: ecs.version
       tag: set_ecs_version
       value: 8.11.0
+  - script:
+      tag: script_add_event_original
+      lang: painless
+      description: Add "event.original"
+      if: ctx.tags != null && ctx.tags.contains('preserve_original_event')
+      source: |
+        def orig = ctx;
+        def stringified_orig = Json.dump(orig);
```

```suggestion
        def stringified_orig = Json.dump(ctx);
```
`orig` refers to the same object, so we might as well use it directly.